_posts/en/newsletters/2021-06-16-newsletter.md

```diff
@@ -0,0 +1,194 @@
+    set their nSequence values to the same values that are required for
+    transactions in certain [multisignature][topic multisignature]-based
+    contract protocols, such as ideas for coinswaps and taproot-enabled
+    LN.  This helps make regular wallet transactions look like contract
+    protocol transactions and vice versa.
+
+    In addition to using nSequence, some contract protocols may also use
+    nLockTime, so Belcher's proposal suggests wallets randomly choose
+    between using either nLockTime or nSequence with 50% probability
+    when both options are available.  Overall, if the proposal is
+    implemented, it will allow users of regular single-sig transactions
+    or uncomplicated multisignatures to join together with users of
+    contract protocols to mutually improve each others' privacy and
+    fungibility.
+
+## FIXME:article
+
+FIXME:harding to rebase on #582 on Tuesday
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- FIXME:harding Rust-Bitcoin 0.26.1 and .2
+
+- FIXME:harding Rust-Bitcoin 0.0.98
+
+- [LND 0.13.0-beta.rc5][LND 0.13.0-beta] is a release candidate that
+  adds support for using a pruned Bitcoin full node, allows receiving
+  and sending payments using Atomic MultiPath ([AMP][topic multipath payments]),
+  and increases its [PSBT][topic psbt] capabilities, among other improvements
+  and bug fixes.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[Rust-Lightning][rust-lightning repo], [libsecp256k1][libsecp256k1
+repo], [Hardware Wallet Interface (HWI)][hwi repo],
+[Rust Bitcoin][rust bitcoin repo], [BTCPay Server][btcpay server repo],
+[Bitcoin Improvement Proposals (BIPs)][bips repo], and [Lightning
+BOLTs][bolts repo].*
+
+- [Bitcoin Core GUI #4][] adds initial support for using [Hardware Wallet
+  Interface (HWI)][topic hwi] external signers via the GUI. Once this feature is
+  finalized, users will be able to use their HWI-compatible hardware wallets
+  directly from the Bitcoin Core GUI.
+
+- [Bitcoin Core #21573][] updates the version of libsecp256k1 included
+  in Bitcoin Core.  Of the changes, the most notable in the use of the
+  optimized modular inverse code described in Newsletters [#136][news136
+  safegcd] and [#146][news146 safegcd].  Performance evaluations posted
+  to the PR found it to speed up old block verification by about 10%.
+
+- [C-Lightning #4591][] adds support for parsing bech32m addresses. C-Lightning
+  will now admit creation of Pay to Taproot (P2TR) (and other v1+ native
+  segwit) outputs when `option_shutdown_anysegwit` is set. Note that P2TR
+  outputs are non-standard and `ANYONE_CAN_SPEND` on mainnet before the Taproot
```

Comment:
Txes with P2TR outputs **are** standard (that's the whole point of this PR).  Txes with P2TR *inputs* are non-standard.  Also I think your formating is confusing anyone-can-spend (lowercase, just a phrase) with `SIGHASH_ANYONECANPAY`.  Instead of saying "anyone can pay", I usually try to be a bit more explicit and say something like, "before the taproot soft fork activates in November, payments sent to a taproot address or any future segwit version address have no security---anyone can spend them."

P.S.  :rofl: youl comment in the commit message.

Reply:
Ah rats, I misunderstood the option to be a configuration option for the node. I hope this is better.